### PR TITLE
Update to spec v1.14, include saves/scenarios install path

### PR DESCRIPTION
**Pull request by plague006**
Documentation changes related to https://github.com/KSP-CKAN/CKAN/pull/1360

### CODE REVIEW

**Comment by Postremus** [comment]
Looks good

File: Spec.md [hunk 1]
```diff
@@ -274,7 +274,7 @@ three source directives:
 In addition a destination directive *must* be provided:
 
 - `install_to`: The location where this section should be installed.
-  Valid values for this entry are `GameData`, `Ships`, `Ships/SPH`(**v1.12**), `Ships/VAB`(**v1.12**), `Tutorial`,
+  Valid values for this entry are `GameData`, `Ships`, `Ships/SPH`(**v1.12**), `Ships/VAB`(**v1.12**), `Tutorial`, `saves/scenarios` (**v1.14**),
```
Comment (pjf) [inline Spec.md]: Oh dear, I've coded it as [just `Scenarios`](https://github.com/KSP-CKAN/CKAN/pull/1360/files#diff-742564cbb6ab84e72817e8b1bd7477a6R277).

**Comment by pjf** [comment]
@plague006 : I've updated #1360 to include the spec bump to v1.14, but we currently differ in terms of the `install_to` keyword. I've used `Scenarios` (which is consistent with `Ships` and `GameData` in terms of capitalisation and plurality).

**Comment by plague006** [comment]
:+1:

**Pull request closed without merging**